### PR TITLE
New, non-underwear hat logo artwork

### PR DESCRIPTION
**Pull request by mairin**
This is the new version of the logo we'd worked on + an update to the README to pull in the new logo.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
bot, add author to whitelist

**Comment by TomSweeneyRedHat** [comment]
bot, test pull request

**Comment by nalind** [comment]
Looks like some of the patches are missing a 'Signed-off-by:' in their commit logs.

File: tests/test_buildah_baseline.sh [hunk 1]
```diff
@@ -74,9 +74,9 @@ scratchmnt=$(buildah mount $newcontainer)
 echo $scratchmnt
 
 ########
-# Install Fedora 26 bash and coreutils
+# Install Fedora 27 bash and coreutils
```
Comment (TomSweeneyRedHat) [inline tests/test_buildah_baseline.sh]: I'm not sure how these two *.sh files got into this commit?  They were part of #349 which I did this weekend and was merged yesterday by @rhatdan?

**Comment by rhatdan** [comment]
@mairin You need to do a

git commit -a --amend -s
git push --force
to add your signature to your PRs.

**Comment by TomSweeneyRedHat** [comment]
#359 replaced this PR and it has been merged.  I'm going to close this one.  @mairin TYVM for the logo updates!

**Pull request closed without merging**